src/clj/rems/api/util.clj

```diff
@@ -49,3 +51,7 @@
   (let [body (json/generate-string {:error (or message "unprocessable entity")})]
     (-> (http-response/unprocessable-entity body)
         (http-response/content-type "application/json"))))
+
+(defn extended-logging [request mutation]
+  (-> (when (some (partial = (:uri request)) (:extended-logging-routes env))
```

Comment:
There may be some cases where this fails. What happens when there query parameters?

If exact string matching is required, it would be useful to use a set. But I think a more realistic version would either
- support regex here
- support matching the request method too or
- automatically log all modifications (`POST`, `PUT`, `DELETE`) in the whole of the API, with one simple config.

Reply:
I agree.

I reckon the need for logging all modifications probably would be a more common scenario than only logging a specific set. And the less common scenario can easily be accommodated by filtering on an application log level.

Reply:
resolved in [317d982](https://github.com/CSCfi/rems/pull/3187/commits/317d982bca638aefb178e89193cfc4ff414227e5)